hypha/templates/base-apply.html

```diff
@@ -102,7 +102,12 @@
                 </div>
 
                 <section class="header__menus header__menus--desktop">
-                    {% include "navigation/primarynav-apply.html" %}
+                    {% if settings.utils.SystemMessagesSettings.nav_content|safe and not request.user.is_authenticated %}
```

Comment:
I think we can skip the "safe" filter in the if statement.

Reply:
For this intermediate version we can skip the "and not request.user.is_authenticated" part as well.

OTF will need this when they launch the new public site. When that happens the "primarynav*.html" will be obsolete.

We need to work on this when we completely remove the public site in any case.

Reply:
Ugh - result of a sloppy copy & paste. Should be all set now.